internal/client.go

```diff
@@ -172,7 +172,7 @@ type (
 		GetWorkflowHistory(ctx context.Context, workflowID string, runID string, isLongPoll bool, filterType enumspb.HistoryEventFilterType) HistoryEventIterator
 
 		// CompleteActivity reports activity completed.
-		// activity Execute method can return acitivity.activity.ErrResultPending to
```

Comment:
II think the real mistake here is `acitivity.activity` should just be `activity` , it should  probably be `[activity.ErrResultPending]`

Reply:
let me double-check

Reply:
indeed, fixed it

Reply:
Thanks!